.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+	"image": "mcr.microsoft.com/devcontainers/python:1-3.11-bookworm",
+	"postCreateCommand": "/bin/bash -c 'python -m pip install poetry && python -m poetry install & git clone https://github.com/reflex-dev/reflex-examples; wait'",
+	"forwardPorts": [3000, 8000],
+	"portsAttributes": {
+		"3000": {
+			"label": "Frontend",
+			"onAutoForward": "notify"
+		},
+		"8000": {
+			"label": "Backend"
```

Comment:
Should we auto-forward the backend port?

Reply:
As far as i know, we have to forward it, and explicitly make it public (since you can't do that in the conf here). It's because each port actually runs on different subdomains through github's proxying infrastructure.